content/en/agent/faq/linux-key-rotation-2024.md

```diff
@@ -0,0 +1,201 @@
+
+{{% /tab %}}
+{{% tab "RedHat/CentOS/SUSE" %}}
+
+Run the following command on the host:
+
+```bash
+$ rpm -qa | grep gpg-pubkey-b01082d3
+```
+
+If the key is trusted, the command has a 0 exit code and outputs:
+
+```
+gpg-pubkey-b01082d3-644161ac
+```
+
+Otherwise, the command returns a non-0 exit code with no output.
+
+Alternatively, check if your repo file contains `https://keys.datadoghq.com/DATADOG_RPM_KEY_CURRENT.public` as one of the `gpgkey` entries. Repo file is usually `datadog.repo` for Agent installations or `datadog-observability-pipelines-worker.repo` for Observability Pipelines Worker. The `CURRENT` key file is updated with the new key as soon as it is in use.
+
+{{% /tab %}}
+{{< /tabs >}}
+
+## The `datadog-signing-keys` package
+
+<div class="alert alert-info"><strong>Note:</strong> This section only applies to DEB Agent package users.</div>
+
+Since Agent v6.31.0 and v7.31.0, all Datadog DEB packages have a soft dependency on the `datadog-signing-keys` package. The following versions of Agent packages have a soft dependency on the `datadog-signing-keys` package version `1.3.1`:
+- datadog-agent, datadog-iot-agent, datadog-heroku-agent, datadog-dogstatsd, datadog-agent-dbg v6.48.1+ & v7.48.1+
+- datadog-fips-proxy v0.5.4+
+- observability-pipelines-worker v1.3.1+
+- datadog-apm-inject v0.10.7+
+- datadog-apm-library-python v1.18.0+
+- datadog-apm-library-java v1.19.1+
+- datadog-apm-library-dotnet v2.36.0+
+- datadog-apm-library-js v4.11.0+
+- datadog-apm-library-all v0.3+
+
+Upon installation, this package:
+
+- Configures APT keys in the `/usr/share/keyrings/datadog-archive-keyring.gpg` keyring and also in `/etc/apt/trusted.gpg.d/datadog-archive-keyring.gpg` when necessary. **This ensures that the upcoming APT repository signing key is trusted.** Using the package `datadog-signing-keys` version 1.3.1 is recommended to ensure preparedness for the upcoming key rotation.
+- Sets up a [`debsig-verify` policy][12] for Datadog packages. This allows you to verify signatures for individual DEB packages locally.
+
+For example, to verify that a locally downloaded DEB package was built and signed by Datadog, run the following command:
+
+  ```bash
+  $ debsig-verify datadog-dogstatsd_7.51.0-1_amd64.deb
+  ```
+
+If the verification is successful, `debsig-verify` exits with status `0` and prints a message: `debsig: Verified package from 'Datadog, Inc.' (Datadog).`. Datadog's DEB packages embed signatures since v6.26.0/7.26.0, so this verification does not work on earlier versions.
```

Comment:
```suggestion
If the verification is successful, `debsig-verify` exits with status `0` and prints a message: `debsig: Verified package from 'Datadog, Inc.' (Datadog).` Datadog's DEB packages embed signatures since v6.26.0/7.26.0, so this verification does not work on earlier versions.
```